src/js/components/Accordion.js

```diff
@@ -1,10 +1,10 @@
 // (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
 
-import React, { Component, PropTypes } from 'react';
```

Comment:
Hey, I believe there is something going on with the re-ordering of the imports. I'm assuming your IDE is sorting them as you save the file?

Can we revert this change?